documentation/service-rules/ecf1/lead-provider-api.md

```diff
@@ -2,237 +2,26 @@
 title: Lead provider API
 ---
 
-Learn about how the payment engine works once it receives declarations from lead providers using the Early Career Framework API.
+Lead providers must use this API to view, submit and update participant data so they receive accurate payments from the Department for Education (DfE) for their ECF-based training.
 
-These are the steps
+Once a participant has been registered to the service by their school, data associated with them becomes available to lead providers via the API.
 
-1. Provider submits a declaration.
-2. The system validates data.
-3. If invalid, errors are flagged up with corrective action suggestions.
-4. If valid, the declaration is created and linked to an output fee calculation.
-5. Service fees and banding are determined and applied.
-6. Final payments are calculated and displayed for contract managers.
+## Contents
 
-## Declarations in a nutshell
+[API versions and updates](#api-versions-and-updates)
 
-Declarations are submitted where there is evidence of a participant's engagement in training for a given milestone period. This is submitted by lead providers via the API and triggers payments to lead providers.
+[How the payment engine works](/service-rules/ecf1/payment-engine)
 
-Declarations serve as the critical "bridge" between proving training engagement and triggering the financial outputs (payments) from the DfE.
+[How participant declarations works](/service-rules/ecf1/participant-declarations)
 
-Declarations are treated in "good faith" meaning that providing the declaration is valid and the participant is eligible for funding then they will be paid.
+## API versions and updates
 
-## Lead providers submitting a declaration
+We regularly make improvements and add new functionality to the API. View our <a href="https://manage-training-for-early-career-teachers.education.gov.uk/api-reference/release-notes.html" target="_blank">release notes</a> to find out about the latest updates.
```

Comment:
although this came from the about the API in ECF1, could we tailor this more to internal docs rather than providers?